src/components/values-field/TextKeyField.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
```

Comment:
@rsumilang use FlatArray field, values-field is not needed

Reply:
@rsumilang we still don't need this one.  flat-array-field provides this already.